inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/animation/ScaleExpandAnimator.kt

```diff
@@ -0,0 +1,74 @@
+package me.saket.inboxrecyclerview.animation
+
+import android.view.View
+import me.saket.inboxrecyclerview.InboxRecyclerView
+import me.saket.inboxrecyclerview.page.ExpandablePageLayout
+
+/**
+ * @param scaling whether to scale items or not, if set to false this animator actually
+ * looks like a simple shared element transition.
+ * @see <a href="https://imgur.com/a/mG0S94t">Watch the examples</a>
```

Comment:
Now that the gifs have been uploaded here, can link to them instead of imgur which may take down the images in the future?